general_autoencoder.py

```diff
@@ -340,24 +371,29 @@ def _fit_from_processed_data(self,
             print('Norm of params: %s' % np.linalg.norm(params['encoder_h0']))
             for epoch in range(self.max_epochs):
                 t0 = time.time()
-                regularization_weighting_for_epoch = self.get_regularization_weighting_for_epoch(epoch)
+
+                regularization_weighting_for_epoch = self.get_regularization_weighting_for_epoch(
+                    epoch,
+                    verbose=verbose)
  
                 self._train_epoch(regularization_weighting_for_epoch)
                     
                 if (epoch % self.num_epochs_before_eval == 0) or (epoch == self.max_epochs - 1):
                     
+                    # regularization weighting is set to 1 just for the purpose of printing out 
+                    # the losses
```

Comment:
i think this will also affect the saved losses (ie, we save a record of losses, and those will now be computed with regularization weighting = 1) which is fine, but something to be aware of. i don't think we would expect the train_mean_combined_loss to decrease monotonically, since that's not what we're actually optimizing.